react/ProductSearchContextProvider.js

```diff
@@ -202,10 +211,13 @@ class ProductSearchContextProvider extends Component {
               loading={loading}
             >
               <Helmet>
-                {titleTag && <title>{titleTag}</title>}
+                {titleTag
+                  ? <title>{titleTag}</title>
```

Comment:
`titleTag` wasn't renamed? Where is it coming from? And also, just use one `title` tag and make the comparison inside

Reply:
This came from the query, the other is from settings

Reply:
Oh, I see now... Confusing the reuse